eglib/configure.ac

```diff
@@ -140,11 +140,11 @@ AC_CHECK_FUNCS(fork execv execve)
 
 AC_ARG_WITH([overridable-allocators], [  --with-overridable-allocators	allow g_*alloc/g_free to call custom allocators set via g_mem_set_vtable])
 
-if test x$with_overridable_allocators == xyes; then
-	AC_DEFINE(ENABLE_OVERRIDABLE_ALLOCATORS,1,[Overridable allocator support enabled])
-  AC_MSG_NOTICE([Overridable allocator support enabled])
-else
+if test x$with_overridable_allocators == xno; then
   AC_MSG_NOTICE([Overridable allocator support disabled])
+else
+  AC_DEFINE(ENABLE_OVERRIDABLE_ALLOCATORS,1,[Overridable allocator support enabled])
+  AC_MSG_NOTICE([Overridable allocator support enabled])
 fi
```

Comment:
If the overhead is significant, we could disable the overridable allocators by default and print an error message in the malloc profiler instructing the user to recompile with overridable allocators enabled to use it. Since this profiler is probably most useful to runtime developers, that's probably fine?

The alternative is disabling the malloc profiler when overridable allocators are disabled, but it would be nice to always build it regardless of the availability of overridable allocators so we can catch breakage early.